### DISCUSSION TITLE
Suggestion for nice feature in the wallet display area.

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#487] [Ideas]
It would be cool if in the wallet section it had a column for the current price in USD and then a total in the bottom as well to show the current value of all coins in USD

I know that would involve some type of API calls to some external location like https://xchforks.com/ to get that info..

But it would be cool to see in Machinaris the total value of my crypto coins without having to manually look it up and run a bunch of calculations.

Juts a thought.

**Comment by guydavis** [comment]
Good idea! I've added an [enhancement](https://github.com/guydavis/machinaris/issues/490) to the backlog.

**Comment by guydavis** [comment]
This will be available in soon to be released version 0.6.9.
